torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1593,16 +1593,17 @@ std::string ProcessGroupNCCL::createLogPrefix() const {
   if (!pg_desc_.empty() && pg_desc_ != "undefined") {
     return c10::str(
         "[PG ID ",
-        uid_,
+        local_uid_,
         "PG GUID ",
-        pg_name_,
+        pg_uid_,
```

Comment:
Is pg_global_uid better? I'm ok with uid too if we take u out of local 

Unlike local id which is a simple counter, this variable has special properties. I want to have a hint about that from the name.

Reply:
I prefer taking out u from local. adding global here seems a little bit duplicate.